### PR TITLE
tickets/PREOPS-94  BNL BPS

### PR DESCRIPTION
**Pull request by womullan**
Added some text as discussed in the meeting last week. Please have a look. I will still add soemthing on Jira planing.

### CODE REVIEW

File: plan.tex [hunk 1]
```diff
@@ -1,5 +1,9 @@
 \section{Planning and team(s) fro DP0} \label{sec:plan}
 
+Planning epics have been (and are) being created in the \href{https://jira.lsstcorp.org/secure/Dashboard.jspa?selectPageId=15608}{PREOPS Jira} project.
+On the dashboard you can see links to the  tickets tagged DP0.1 and DP0.2.
```
Comment (timj) [inline plan.tex]: label not tag

File: plan.tex [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+
+We will have regular (every other week for now) DP0 meetings see \url{https://confluence.lsstcorp.org/display/LSSTOps/Data+Production+Meetings}.
```
Comment (timj) [inline plan.tex]: There has to be punctuation between "meetings" and "see" (maybe use parentheses around "see url").

File: plan.tex [hunk 3]
```diff
@@ -57,15 +70,32 @@ \subsubsection{Middleware}
 we will have to do on IDF.
 
 \paragraph{Workflow} needs to be functioning at scale for DP0.2, ideally we should basic workflow
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also. We shall evaluate BPS as an option later in 2020 (DP-MW-M07). See \citeds{LDM-636} \citeds{LDM-633} \citeds{DMTN-123}.
```
Comment (timj) [inline plan.tex]: We prefer one sentence per line to make diffs easier.

Punctuation between the LDM mentions?

File: plan.tex [hunk 4]
```diff
@@ -57,15 +70,32 @@ \subsubsection{Middleware}
...
+BPS translates the quantum graph to DAGMan for execution on HTCondor and submits the jobs.
+Most work has gone into the graph and execution.
+
+As part of our march toward a potential more DOE oriented Data Facility BNL will be part of the the pre operations team to experiment with PanDA as an environment to monitor and control our HTCondor jobs.
```
Comment (timj) [inline plan.tex]: "DOE-oriented"
Comma between "Facility" and "BNL".
Does PanDA uses HTCondor? Probably easier to say "control our processing jobs"

File: plan.tex [hunk 5]
```diff
@@ -57,15 +70,32 @@ \subsubsection{Middleware}
...
+This is a slightly parallel effort to constriction attempting to take advantage of a large scale
+existing set of tools for large scale job execution. In an ideal world the quantum graph translation
```
Comment (timj) [inline plan.tex]: This has switched back to hard wraps mid sentence.

Typo: constriction.
Probably shouldn't say "large scale" twice in one sentence.

File: plan.tex [hunk 6]
```diff
@@ -57,15 +70,32 @@ \subsubsection{Middleware}
...
+of BPS would feed into a PanDA system to execute (retry etc) our jobs.
```
Comment (timj) [inline plan.tex]: This is an area of investigation because I don't think PanDA accepts DAGMan at the moment.

File: tables/teams.tex [hunk 7]
```diff
@@ -5,50 +5,50 @@
...
+{3.2l}&{Infrastructure and Support}&{Wide Area Network Architect}&{Familiar with WAN implementation technologies generally available in the networks supporting the Rubin Observatory. Familiar with technology roadmap of the ESNet WAN provider. Synthesizes and evolves network techniques and provisioning supporting the Rubin Observatory mission, as network technology evolves. Drawn from staff of WAN groups but explicitly supported by and work in the context of the Rubin Observatory.}&{Fermilab}&{0.38} \\ \hline
+{3.2r}&{Infrastructure and Support}&{Data Wrangler - IN2P3}&{The data wrangler ensures that data (science raw data, calibration data, data products, etc.) is replicated at IN2P3 and data products resulting from the local processing performed at IN2P3 are replicated to the USDF They also ensure that the data archived at IN2P3 and needed for the annual processing are recalled from tape on time for the image processing tasks to be performed. They ensure that the tools and systems used for replicating data at IN2P3 are operational. This role needs tight coordination with the team at the USDF that is responsible for data distribution.}&{IN2P3}&{0.35} \\ \hline
```
Comment (timj) [inline tables/teams.tex:11]: These hyphens should all be at least en-dashes.

File: plan.tex (hunk 1 above)
Comment (womullan) [reply plan.tex]: ok

File: tables/teams.tex (hunk 7 above)
Comment (womullan) [reply tables/teams.tex:11]: this is generated from a google sheet .. i guess i can make it do a(nother) substitution but perhpas not today

File: plan.tex (hunk 6 above)
Comment (womullan) [reply plan.tex]: noted

File: plan.tex (hunk 4 above)
Comment (womullan) [reply plan.tex]: ok

File: plan.tex (hunk 3 above)
Comment (womullan) [reply plan.tex]: ok

File: plan.tex (hunk 5 above)
Comment (womullan) [reply plan.tex]: ok

File: plan.tex [hunk 8]
```diff
@@ -57,15 +70,33 @@ \subsubsection{Middleware}
...
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also.
+We shall evaluate BPS as an option later in 2020 (DP-MW-M07).
+See \citeds{LDM-636}, \citeds{LDM-633}, \citeds{DMTN-123}.
+BPS translates the quantum graph to DAGMan for execution on HTCondor and submits the jobs.
+Most work has gone into the graph and execution.
+
+As part of our march toward a potential more DOE oriented Data Facility, BNL will be part of the the pre operations team to experiment with PanDA as an environment to monitor and control our processing jobs.
```
Comment (hsinfang) [inline plan.tex]: > part of the the pre operations team

Remove the extra "the"

File: plan.tex [hunk 9]
```diff
@@ -57,15 +70,33 @@ \subsubsection{Middleware}
 we will have to do on IDF.
 
 \paragraph{Workflow} needs to be functioning at scale for DP0.2, ideally we should basic workflow
-early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03)
+early on (milestone DP-MW-M-02). Then more tooling such as restarting failed jobs (DP-MW-M-03).
+
+From the construction side we have BPS as a deliverable which may be useful on IDF also.
+We shall evaluate BPS as an option later in 2020 (DP-MW-M07).
+See \citeds{LDM-636}, \citeds{LDM-633}, \citeds{DMTN-123}.
+BPS translates the quantum graph to DAGMan for execution on HTCondor and submits the jobs.
```
Comment (hsinfang) [inline plan.tex:78]: Did we give up using CWL or other standard workflow format?

File: plan.tex [hunk 10]
```diff
@@ -57,15 +70,33 @@ \subsubsection{Middleware}
...
 \subsubsection{ Pipelines }
 For DP0.2 we need a Gen3 version of the pipelines to process the dataset. This will have to run at scale for PDR2 or DC2. There may be several runs for quality purposes.
-Fractional FTE from the Pipelines will provide help in pipeline configuration, data repo preparation, workflow consulting, science verification, data model documenting, troubleshooting, and liaisoning.
+Fractional FTE from the Pipelines will provide help in pipeline configuration, data repo preparation, workflow consulting, science verification, data model documenting, troubleshooting, and liaising.
 {\bf Yusra will provide more info here.}
 
-\subsubsection{ IN2P3}
+\subsubsection{ IN2P3}\label{sec:in2p3}
 IN2P3 will contribute in Qserv and pipelines. {\bf Fabio will provide more information here.}
+They bring experience running Gen3 workflows. The real interest with IN2P3 is to run remote jobs thus emulating the eventual operational DRP runs. This may be difficult to achieve in FY21 but we should make it a milestone for FY22.\footnote{Tim, Fabio we should set a date for this}.
```
Comment (hsinfang) [inline plan.tex]: In FY21 we might try to duplicate the same processing on IDF & IN2P3.

File: plan.tex (hunk 8 above)
Comment (womullan) [reply plan.tex]: ok

File: plan.tex (hunk 10 above)
Comment (womullan) [reply plan.tex]: ok

File: plan.tex (hunk 9 above)
Comment (womullan) [reply plan.tex:78]: no I can add CWL

**Pull request merged**